src/app/node-data/basic/provider/vsphere/component.ts

```diff
@@ -121,7 +121,8 @@ export class VSphereBasicNodeDataComponent extends BaseFormValidator implements
     }
   }
 
-  private _setDefaultTemplate(os: OperatingSystem): void {
+  private _setDefaultTemplate(os: OperatingSystem = undefined): void {
```

Comment:
If this parameter is no longer used, I'd remove it.

Reply:
It's used in line 108.